src/test/java/com/mealmentor/enterprise/EnterpriseApplicationTests.java

```diff
@@ -4,7 +4,6 @@
 import com.mealmentor.enterprise.dto.*;
 import com.mealmentor.enterprise.service.IMealPlanService;
 import com.mealmentor.enterprise.service.MealPlanServiceStub;
-import lombok.SneakyThrows;
```

Comment:
Minor thing but I removed all unused imports in the project. Always a good idea to go through the warnings the IDE gives you to see what minor changes you can make to your code.